llvm/utils/repo/Makefile

```diff
@@ -40,17 +49,18 @@ libunwind.a:
 		-D utils_dir=$(LLVM_REPO)/llvm/utils/repo                     \
 		-D CMAKE_INSTALL_PREFIX=$(LLVM)                               \
 		-D musl_install=$(MUSL)                                       \
-		../libunwind &&                                               \
-	make -j 8 &&                                                          \
+		../libunwind
+	make -j 8
 	make install
 
-libcompiler-rt.a:
-	if [ -d "$(LLVM_REPO)/build_libcompiler-rt" ];then                    \
-		rm -rf $(LLVM_REPO)/build_libcompiler-rt;                     \
-	fi
-	cd $(LLVM_REPO)  &&                                                   \
-	mkdir build_libcompiler-rt && cd build_libcompiler-rt  &&             \
-	export REPOFILE=$(LLVM_REPO)/build_libcompiler-rt/libcompiler-rt.db &&\
+.ONESHELL:
+libcompiler-rt.a $(CLANG_RT)/lib/linux/clang_rt.crtbegin-x86_64.t \
+	$(CLANG_RT)/lib/linux/clang_rt.crtend-x86_64.t \

```

Comment:
This path ($(CLANG_RT)/lib/linux/) appears _a lot_. Would adding a variable for it make things easier to read?

Reply:
Good point, fixed in the commit https://github.com/SNSystems/llvm-project-prepo/commit/841f014334761d2b2bf926c7063a4085f53f9cf1.